CMakeLists.txt

```diff
@@ -1,31 +0,0 @@
-cmake_minimum_required (VERSION 3.0)
```

Comment:
if you prefer to have a home brew Makefile for making the whole test suite like what runtime verification has that's OK. But I think it is important to have build system for the whole suit.

Reply:
As above, I added a Makefile for that.